newsletters/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class department(models.Model):
+    '''
+    Stores the detail of department linked with its newsletters
+    '''
+    department = models.CharField(max_length=50,
+                                  blank=False,
+                                  null=False)
+
+    def __unicode__(self):
+        return "%s" % (self.department)
+
+
+class papers(models.Model):
+    '''
+    Stores the pdfs of newsletters
+    '''
+    name = models.CharField(null=False,
+                            max_length=25)
+    uploaded_by = models.ForeignKey(User)
```

Comment:
I think this field must be linked to `FacultyDetail` model.

Reply:
Yes sir, I did not notice this existing model. Will update this.